cli.js

```diff
@@ -62,19 +65,43 @@ if (!tasks || Object.keys(tasks).length < 1) {
 	process.exit(0);
 }
 
+// get package.json descriptions value
+if (argv.descriptions) {
+	try {
+		descriptions = { exports: {} };
+		require("pkginfo")(descriptions, { dir: cwd, include: ["ntl"] });
+		descriptions = descriptions.exports.ntl.descriptions;
+	} catch (e) {
+		error(e, "No descriptions for your npm scripts found");
+	}
+}
+
 // defines the items that will be printed to the user
 const input = (argv.info
 	? Object.keys(tasks).map(i => ({ name: `${i}: ${tasks[i]}`, value: i }))
-	: Object.keys(tasks)
+	: argv.descriptions
+		? Object.keys(tasks).map(i => {
+			return {
+				name: `${i}: ${descriptions[i]}`,
+				value: i
+			}
+		})
+		: Object.keys(tasks)
 ).filter(
 	// filter out prefixed tasks
 	i =>
 		argv.all
 			? true
-			: ["pre", "post"].every(prefix => argv.info
+			: ["pre", "post"].every(prefix => argv.info || argv.descriptions
 					? i.name.slice(0, prefix.length) !== prefix
 					: i.slice(0, prefix.length) !== prefix
 			)
+).filter(
```

Comment:
I prefer filtering out the tasks which doesn’t have a description. Are you okay with this?

Reply:
Yeah. I really need to filter out tasks which aren’t meant to be used by the developer. That would make it easier to know which are the main tasks of this beast:
https://github.com/micromata/Baumeister/blob/4b0ca35e774b323fa84bb1ad71153deb7d07d340/package.json#L19-L46

So I need to introduce a new option like:

> -l, --limit         Limits output to scripts with a description      [boolean]

Which means without this option I would have a mixed output like that:

Reply:
Just recognized that we should use another delimiter between script name and description. Too much colons in here 😆

Reply:
Change the delimiter with e3dfaee.

Reply:
I’ve implemented it with the `--limit` option for now. See 6d4d7b1.

Reply:
let's make it `--descriptions-only` with the short alias `-o`

Reply:
Done.